### PR TITLE
Install quick wins

### LINKED ISSUE #9782: Quick install wins
Dev Rel listed some quick wins [here](https://www.notion.so/rasa/Rasa-Installation-Friction-Log-September-2021-ca44ca1c239d4087b91038441b770208)

### PR DESCRIPTION
**Pull request by TyDunn**
**Proposed changes**:
- Closes #9782

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: changelog/9780.doc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Updates quick install documentation with optional venv step, better pip install instructions, & M1 warning
```
Comment (wochinge) [inline changelog/9780.doc.md:1]: optional: add link to the "install documentation" in the changelog to make it easier discoverable for our users

File: docs/docs/installation.mdx [hunk 2]
```diff
@@ -18,22 +18,77 @@ your machine and continue development locally.
...
+**You are now ready to go! So what's next? You can create a new project by running:**
 
 ```bash
 rasa init
 ```
 
 You can learn about the most important Rasa commands in the [Command Line Interface](./command-line-interface.mdx).
 
+:::note
+Due to lack of official TensorFlow support for the Apple M1, Rasa Open Source is unable to train a model using M1.
```
Comment (wochinge) [inline docs/docs/installation.mdx:90]: ?
```suggestion
Due to lack of official TensorFlow support for the Apple M1, Rasa Open Source is currently unable to train a model using M1.
```

**Review by wochinge — COMMENTED** [review]
Left some comments 👍🏻 

if for one reason or another at some point the Docs tests break due to something related to `DeclarativeMeta` while I'm gone tomorrow: Please apply this commit then https://github.com/RasaHQ/rasa/commit/4179876e36c8bef38074561128407cff7e078a31

File: docs/docs/installation.mdx [hunk 3]
```diff
@@ -18,22 +18,77 @@ your machine and continue development locally.
...
+  source ./venv/bin/activate
+  ```
+
+  </TabItem>
+  <TabItem value="windows">
+
+  Create a new virtual environment by choosing a Python interpreter and making a `.\\venv` directory to hold it:
+
+  ```bat
+  C:\> python3 -m venv ./venv
```
[the remark this answers is not in the document]
Comment (TyDunn) [reply docs/docs/installation.mdx:59]: I duplicated this section from the existing docs 🤔

File: changelog/9780.doc.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Updates quick [install documentation](https://rasa.com/docs/rasa/installation#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```
Comment (TyDunn) [inline changelog/9780.doc.md:1]:
```suggestion
Updates [quick install documentation](https://rasa.com/docs/rasa/installation#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```

File: docs/docs/installation.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (rctatman) [reply docs/docs/installation.mdx:59]: further down on the same page it's ./venv & I don't think `python3` will work on windows (which would be .\)

**Review by rctatman — APPROVED** [review]
added suggestion

File: docs/docs/installation.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (TyDunn) [reply docs/docs/installation.mdx:59]: @rctatman I'm not sure I follow. What are you suggesting here?

File: changelog/9780.doc.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Updates [quick install documentation](https://rasa.com/docs/rasa/installation#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```
Comment (wochinge) [inline changelog/9780.doc.md:1]: This uses relative links (e.g. better if we consider publishing the docs under a different URL at some point) and _I think_ also considers the used version
```suggestion
Updates [quick install documentation](installation.mdx#quick-installation) with optional venv step, better pip install instructions, & M1 warning
```

**Pull request merged**